apps/mttr-application/.env.example

```
@@ -0,0 +1,65 @@
+# (~60s) — otherwise pg hands out a connection the server has already
+# reaped and the next query fails with ECONNRESET.
+DB_POOL_IDLE_MS=30000
+# How long to wait for a free client from the pool before erroring out.
+# Prevents requests from hanging indefinitely when the DB is unreachable.
+DB_POOL_CONNECT_MS=10000
+# Postgres-side per-query deadline (SET statement_timeout, ms). Any query
+# running longer than this is cancelled server-side and its pool client
+# returns to the pool. Must sit above every legitimate query duration —
+# aggregation ~10s, retention ~30s per full run; 2 min gives comfortable
+# headroom. Raise if the workload ever grows past that.
+DB_STATEMENT_TIMEOUT_MS=120000
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DB_STATEMENT_TIMEOUT_MS key should go before the DB_USER key

(UnorderedKey)